### PR TITLE
Added display:blocks to spans in tooltip/infotip

### PR DESCRIPTION
**Pull request by agliga**
## Description

Changed all tooltip and infotips to support span content. Mainly added display:blocks in span content to look like a div

## Context

In order to fix infotip nested inside a paragraph, no block level elements are allowed. This includes `<div>`, `<h1-6>` or other `<p>` tags. However, styling an inline element to block level element is allowed. Since the infotip needs to live inside the `<p>` element, need to change all div's to spans and then has to add styles to make it look like it initially did.
Since infotip uses tooltip, the Ebayui change was to change the tooltip elements to use spans. This is why the styles cross into the other ones as well. 

## References

Fixes eBay/ebayui-core#729

## Screenshots

image

### CODE REVIEW

**Comment by seangates** [comment]
So, since this will make the block elements deprecated, please indicate that in the LESS source.

**Comment by agliga** [comment]
> So, since this will make the block elements deprecated, please indicate that in the LESS source.

So block elements should still be allowed in non `<p>` scenarios. 
Should I update and add a `<p>` scenario?

**Comment by seangates** [comment]
Yes, we need to inform the developers of the correct way to implement.

File: docs/_includes/common/tooltip.html [hunk 1]
```diff
@@ -110,9 +112,70 @@ <h3 class="infotip__heading">Infotip</h3>
             </div>
         </div>
     </div>
+    </span>
```
Comment (seangates) [inline docs/_includes/common/tooltip.html]: Is this an errant `</span>`? It popped up in the most recent change, but without a corresponding opening tag.

File: docs/_includes/common/tooltip.html [hunk 2]
```diff
@@ -110,9 +112,70 @@ <h3 class="infotip__heading">Infotip</h3>
...
 </span>
     {% endhighlight %}
 
+    <p>
+        In case you want to nest the infotip inside a paragraph element, you need to make sure that
+        there are no block level elements, such as div, h1-6, or p elements.
```
Comment (seangates) [inline docs/_includes/common/tooltip.html]: Can you format them?

```such as `<div>`, `<h1-6>`, or `<p>` elements.```

File: docs/_includes/common/tooltip.html (hunk 1 above)
Comment (agliga) [reply docs/_includes/common/tooltip.html]: Good catch, was an issue with copy and pasting 👍

File: docs/_includes/common/tooltip.html (hunk 2 above)
Comment (agliga) [reply docs/_includes/common/tooltip.html]: Added `<span class="highlight"></span>`

**Pull request merged**